src/Maker/MakeEntity.php

```diff
@@ -35,49 +60,706 @@ public static function getCommandName(): string
+
+        // help the user select the type
+        if ('relation' === $type) {
+            $type = $this->askRelationType($io, $generatedEntityClass, $targetEntityClass);
+        }
+
+        $askFieldName = function (string $targetClass, string $defaultValue) use ($io) {
+            return $io->ask(
+                sprintf('New field name inside %s', Str::getShortClassName($targetClass)),
+                $defaultValue,
+                function ($name) use ($targetClass) {
+                    // it's still *possible* to create duplicate properties - by
+                    // trying to generate the same property 2 times during the
+                    // same make:entity run. property_exists() only knows about
+                    // properties that *originally* existed on this class.
+                    if (property_exists($targetClass, $name)) {
+                        throw new \InvalidArgumentException(sprintf('The "%s" class already has a "%s" property.', $targetClass, $name));
+                    }
+
+                    return Validator::validateDoctrineFieldName($name, $this->getRegistry());
+                }
+            );
+        };
+
+        $askIsNullable = function (string $propertyName, string $targetClass) use ($io) {
+            return $io->confirm(sprintf(
+                'Is the <comment>%s</comment>.<comment>%s</comment> property allowed to be null (nullable)?',
+                Str::getShortClassName($targetClass),
+                $propertyName
+            ));
+        };
+
+        $askOrphanRemoval = function (string $owningClass, string $inverseClass) use ($io) {
+            $io->text([
+                sprintf(
+                    'A <comment>%s</comment> becomes "orphaned" if it is removed from its related <comment>%s</comment>.',
+                    Str::getShortClassName($owningClass),
+                    Str::getShortClassName($inverseClass)
+                ),
+                sprintf(
+                    'For example: <comment>$%s->remove%s($%s)</comment>',
+                    Str::asLowerCamelCase(Str::getShortClassName($inverseClass)),
+                    Str::asCamelCase(Str::getShortClassName($owningClass)),
+                    Str::asLowerCamelCase(Str::getShortClassName($owningClass))
+                ),
+                '',
+                sprintf(
+                    'NOTE: If a <comment>%s</comment> should be allowed to *change* from one <comment>%s</comment> to another, answer "no".',
+                    Str::getShortClassName($owningClass),
+                    Str::getShortClassName($inverseClass)
+                ),
+            ]);
+
+            return $io->confirm(sprintf('Do you want to automatically delete orphaned <comment>%s</comment> objects (orphanRemoval)?', $owningClass), false);
+        };
+
+        $setMapInverseSide = function (EntityRelation $relation) {
+            $relation->setMapInverseRelation(!$this->isClassInVendor($relation->getInverseClass()));
```

Comment:
So, it is even impossible to disable the generation of the bidirectional relation here, as you don't ask 😢

Reply:
That's correct :). I need some convincing that there is a downside before we add more options